doc/INSTALL_ru-RU.md

```diff
@@ -0,0 +1,101 @@
+# Выберите свой способ установки
+
+1. [Установка из бинарных файлов](#Install-from-binaries)
+
+   Это самый простой способ, который рекомендуется для большинства пользователей.
+
+2. [Сборка из исходного кода](#Build-from-source)
+
+   Таким образом, вы можете получить самые последние неизданные функции и подготовиться к участию. Это требует дополнительных шагов, и это единственный выбор, если ваша ОС официально не поддерживается.
+
+3. [Использовать Артефакты](#Использовать-Артефакты)
```

Comment:
I don't think "Artifacts" should be translated, it's like a proper noun on GitHub.

Reply:
Okay, got it!

Reply:
> I don't think "Artifacts" should be translated, it's like a proper noun on GitHub.

No, "Artifacts" can be translated. For example https://ru.werf.io/documentation/configuration/stapel_artifact.html

Reply:
> No, "Artifacts" can be translated. For example https://ru.werf.io/documentation/configuration/stapel_artifact.html

OK